src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralYield.java

```diff
@@ -0,0 +1,19 @@
+/*
+WhitespaceAfter
+tokens = LITERAL_YIELD
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespaceafter;
```

Comment:
You need comment like 
https://github.com/checkstyle/checkstyle/blob/a9e53970c2809a408ce05f24c7c1a380923a8f11/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterSwitchStatements.java#L7

To let CI pass

Reply:
Done